packages/jest-test-result/src/types.ts

```diff
@@ -40,7 +40,9 @@ export type Bytes = number;
 
 export type Milliseconds = TestResult.Milliseconds;
 
-export type AssertionResult = TestResult.AssertionResult;
+export type AssertionResult = TestResult.AssertionResult & {
+  failureDetails: Array<unknown>;
```

Comment:
I did this during the rebase to limit the amount of explosions - looking around what happened, I think it's logical to move this change into `TestResult.AssertionResult`, which I'll do now to see if it causes errors anywhere else.